crates/sparrow-backend/src/logical_to_physical.rs

```diff
@@ -296,6 +359,24 @@ impl LogicalToPhysical {
     }
 }
 
+fn logical_to_physical_literal(
```

Comment:
I'll probably file an issue -- I think we can replace our `ScalarValue` with an arrow `Scalar(ArrayRef)`, at which point we can have Arrow serialize them, delete all that code, and make it a lot easier to represent literals and scalars. We'll see.

Reply:
Already had it. https://github.com/kaskada-ai/kaskada/issues/787